app/src/main/java/com/yourbcabus/android/yourbcabus/BusListActivity.kt

```diff
@@ -296,6 +297,20 @@ class BusListActivity : AppCompatActivity() {
                 detailString = "Arrived at BCA"
             }
 
+            if (item.departure != null) {
+                val cal = GregorianCalendar()
```

Comment:
How are you planning on implementing timezone support?

Reply:
In my tests, this defaults to the user's local timezone. Given that `departure` only contains the number of minutes past midnight, this behavior seems fine.

Here are some examples (with an expanded date format to show more information):
![image](https://user-images.githubusercontent.com/36865364/96367517-53588a00-111c-11eb-8318-8b4cc4c522d7.png)
This is the date with my default time zone (ie. `America/New_York`).

![image](https://user-images.githubusercontent.com/36865364/96367527-5fdce280-111c-11eb-95b9-bc086ed2a89c.png)
This is when I manually set my device to `America/Los_Angeles`.

Reply:
Alright, that should be fine.

The behavior of the web app and the iOS app displays the departure time in Eastern time and converts it to the user's time zone. It differs from the Android app but we can always fix it in the future.